.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-3.1.2
```

Comment:
Always update the version to match the version in the gemfile.

Reply:
Alright @bellom It was during my deployment to render, that made me change it to 3.1.2 because the ruby  version existing in render is 3.1.2